src/main/io/dashboard.c

```diff
@@ -421,7 +421,7 @@ static void showGpsPage(void)
     i2c_OLED_set_line(bus, rowIndex);
     i2c_OLED_send_string(bus, lineBuffer);
 
-    tfp_sprintf(lineBuffer, "ERRs: %d", gpsData.errors, gpsData.timeouts);
+    tfp_sprintf(lineBuffer, "ERRs: %d", gpsData.errors);
```

Comment:
Hmm, or should this be `tfp_sprintf(lineBuffer, "ERRs: %d / %d", gpsData.errors, gpsData.timeouts)` or similar?

Reply:
That's how the code was written originally.  Commit 4746b336b9870bf98f0df237438ec4db9bbb7c69 split it into two separate calls, but neglected to make the corresponding change to the argument list.  gpsData.timeouts is now printed separately (line 434 in the current revision).  So I think the right fix is simply to remove the unused argument.

Reply:
By the way, GCC offers a "format" function attribute which would enable it to warn about these kinds of errors.  It's used like this:

`extern int
my_printf (void *my_object, const char *my_format, ...)
      __attribute__ ((format (printf, 2, 3)));
`

Should we start using it?  It also warns about type mismatches between the format string and the arguments.  The documentation is [here](https://gcc.gnu.org/onlinedocs/gcc-9.3.0/gcc/Common-Function-Attributes.html#Common-Function-Attributes).

Reply:
Good idea to add these warnings.
However, the plan is to move to https://github.com/mpaland/printf/. There are a couple of pull requests by @ledvinap into https://github.com/mpaland/printf/ pending, and the hope is that these will be merged at some time. But seeing how long this is taking we might as well move now, and cherry-pick the contents of the pull requests for now.